.github/CODEOWNERS

```
@@ -43,7 +43,7 @@
 /tests/tests_fabric                           @awaelchli @carmocca @justusschock
 
 # Lightning App
-/src/lightning_app                          @tchaton @lantiga @awaelchli @hhsecond @ethanwharris
+/src/lightning/app                          @tchaton @lantiga @awaelchli @hhsecond @ethanwharris
```

```suggestion
/src/lightning_app                          @tchaton @lantiga @awaelchli @hhsecond @ethanwharris
```
lets edit all at once...